examples/walkthrough/k8s201.md

```diff
@@ -147,7 +241,8 @@ spec:
         - containerPort: 80
 ```
 
```

Comment:
Can we add a pointer here to the "Container Probes" section in the "Life of a Pod" page? 

For more information about health checking, see [Container Probes](../../docs/pod-states.md#container-probes).